src/org.xtuml.bp.mc.masl/build_settings/build_setting.properties

```diff
@@ -0,0 +1,9 @@
+# These settings are used by the MC-3020 builder
+
+build_command =
+marking_file_location = defaults/marking

```

Comment:
Ctrl-Ms at the end of a bunch of lines here???

Reply:
done